main.cpp

```diff
@@ -140,8 +265,25 @@ struct SimpleLooper
     float loopEndPoint = 10.f;
     bool shouldLoop = true;
 
-    SimpleLooper(int audioClipList);
+    //SimpleLooper(int audioClipList); this was the original member function, but I am not sure how to call it below. I know we need to specify the type, but since there is no type, how would this work? I am adding a new function for now. ALSO, this function was causing problems with declaring a type, comment down below
+    int createLoopPoint(int startPoint, int endPoint);
+    void startLooping();
 };
+
+int SimpleLooper::createLoopPoint(int startPoint, int endPoint)
+{
+    SimpleLooper myLoop; // When I had SimpleLooper(int audioClipList) above, it was causing "no matching constructor for initialization of myLoop"
```

Comment:
Yeah, you need to provide that argument since that's the only way to construct that `SimpleLooper`.  Constructor arguments are covered later.